### PR TITLE
Added GDBrowser Presence

### PR DESCRIPTION
**Pull request by etstringy**
Presence for [GDBrowser](https://gdbrowser.com) written in JavaScript

Screenshots: 
image

image

image

image

### CODE REVIEW

File: GDBrowser/dist/presence.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+presence.on("UpdateData", async () => {
+    let presenceData = {
+        largeImageKey: "coin"
+    }
+
+    const q = new URLSearchParams(window.location.search);
+
+    if (window.location.href.includes("gdbrowser.com")) {
+            console.log("PreMID Presence Active.")
```
Comment (eggsywashere) [inline GDBrowser/dist/presence.js]: Better remove this one :^)

File: GDBrowser/dist/presence.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+                // presenceData.details = "Viewing the homepage";
+            }
+
+
+            // other stuff 
+            if (window.location.pathname.toLowerCase() === "/iconkit") {
+                presenceData.details = "In the iconkit";
+            }
+
+            // if (window.location.pathname.toLowerCase() === ("/search")) {
```
Comment (eggsywashere) [inline GDBrowser/dist/presence.js]: Also remove commented/unneeded code.

File: GDBrowser/dist/presence.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+                    presenceData.details = "Searching for levels";
+
+                    // Map Pack
+                    if(q.get("mappack") === 1){
+                        presenceData.state = "Viewing a map pack";
+
+
+                    // Quick Search
+                    } else if(q.get("type") === "recent"){
```
Comment (eggsywashere) [inline GDBrowser/dist/presence.js]: Please use `switch`, it's easier. Here's an easy example:
```js
switch(q.get("type")) {
   case "recent":
      pdata.details = "Recent";
      break;
   case "something":
     pdata.details = "Something";
     break;
}```

File: GDBrowser/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+    "author": {
+      "name": "Stringy",
+      "id": "577743466940071949"
+    },
+    "service": "GD Browser",
```
Comment (Bas950) [inline GDBrowser/dist/metadata.json]: Service name must be the same as folder name, so change either one

File: GDBrowser/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    "description": {
+      "en": "Browse all of Geometry Dash's online features, right from this handy little website! Levels, profiles, leaderboards, comments, and more!"
+    },
+    "url": "gdbrowser.com",
+    "version": "0.1",
+    "logo": "https://gdbrowser.com/assets/coin.png",
```
Comment (Bas950) [inline GDBrowser/dist/metadata.json]: Version should be 1.0 on first release
Upload logo to imgur

**Comment by etstringy** [comment]
Right, I'll get these changes done. Especially that horrible elseif.

**Comment by etstringy** [comment]
Ok, It should be all good now. Lmk if anything still needs changing

File: GDBrowser/dist/presence.js [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+            // If the url has numbers at the end (/)
+            if (!isNaN(window.location.pathname.toLowerCase().replace("/", "")) && window.location.pathname.toLowerCase() !== "/"){
+                LevelData()
+            }
+
+            if (window.location.pathname.toLowerCase() === "/daily") {
+                LevelData()
+            }
+            
+            if (window.location.pathname.toLowerCase() === "/weekly") {
```
Comment (eggsywashere) [inline GDBrowser/dist/presence.js]: You can also use or (`||`) operator to check for multiple things in a single if statement. If you're going to use the same function for all these different pages, you can just do `if (one || two && two2 || three || goesOn) { }`

Comment (etstringy) [reply GDBrowser/dist/presence.js]: Ahh, ok. Gotcha

Comment (eggsywashere) [reply GDBrowser/dist/presence.js]: Sorry for the late addition, I was unable to check the code more carefully 😓

**Pull request merged**